lib/datadog/lambda/version.rb

```diff
@@ -12,7 +12,7 @@ module Datadog
   module Lambda
     module VERSION
       MAJOR = 2
-      MINOR = 16
+      MINOR = 17
```

Comment:
Are there breaking changes in this PR, if so, we should bump `MAJOR` instead. Otherwise a `MINOR` bump is fine.

Reply:
No breaking changes in this PR :-)